tsconfig.json

```diff
@@ -1,18 +1,23 @@
 {
-    "version": "1.8.0",
+    "compileOnSave": true,
     "compilerOptions": {
-        "outDir": "build",
-        "target": "es6",
-        "module": "commonjs",
-        "moduleResolution": "node",
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
-        "sourceMap": true,
+        "module": "commonjs",
+        "target": "es6",
         "noImplicitAny": true,
-        "declaration": true
+        "sourceMap": true,
+        "outDir": "build",
+        "moduleResolution": "node",
+        "typeRoots": [
+            "node_modules/@types"
+        ]
     },
+    "include": [
+        "src/**/*.ts"
+    ],
     "exclude": [
-        "build",
-        "node_modules"
+        "node_modules",
+        "build"
     ]
```

Comment:
> whats point of having both include and exclude?

True, I will update it to have include only.

> Also your typeroots value is by default - there is no need to set it again

By default it is set loosely, so it can pick d.ts files from there and other places as well, by setting it explicitly, we ensure it will picky typings only from there, so no chance of definition clashing.